pyproject.toml

```diff
@@ -1,8 +1,73 @@
+[project]
+name = "calcos"
+description = "Calibration software for COS (Cosmic Origins Spectrograph)"
+requires-python = ">=3.8"
+authors = [
+    { name = "Phil Hodge" },
+    { name = "Robert Jedrzejewski" },
+]
+classifiers = [
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: C",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+dependencies = [
+    "astropy>=5.0.4",
+    "numpy",
+    "scipystsci.tools>=4.0.0",
+]
+dynamic = [
+    "version",
+]
+
+[project.readme]
+file = "README.md"
+content-type = "text/markdown"
+
+[project.scripts]
+calcos = "calcos:main"
+
```

Comment:
Is it better to do this with entry_points?

Reply:
It is, counterintuitively. The `project.scripts` table is analogous to `entry_points.console_scripts` from `setup.cfg` (the old `scripts` parameter of `setup()` is not included in PEP621 in favor of using entry points instead).
https://peps.python.org/pep-0621/#entry-points 
> There are three tables related to entry points. The `[project.scripts]` table corresponds to the `console_scripts` group in the [entry points specification](https://packaging.python.org/specifications/entry-points/). The key of the table is the name of the entry point and the value is the object reference.

Reply:
Ah, OK!!  That is confusing, isn't it?